Pipfile

```diff
@@ -55,10 +55,10 @@ jiwer = "*"
 fuzzywuzzy = "*"
 python-levenshtein = "*"
 asrtoolkit = "*"
+wordcloud = "*"
 
 [requires]
 python_version = "3"
 
-[packages.uvicorn]
-extras = [ "standard",]
-version = "==0.11.8"
+[packages]
+uvicorn = {extras = [ "standard",], version = "==0.11.8"}
```

Comment:
Is this the source of the error about seeing packages twice?

Reply:
I'm using the new pipfile/requirements from your cleaned up version, so it should render this moot